e2e2/test/cases/bert-inference/bert_inference_test.go

```diff
@@ -0,0 +1,75 @@
+package bert_inference
```

Comment:
can we just call this package `inference`? we realistically may add other inference cases that aren't BERT

Reply:
Yeah sure thing. Done.